timecache/first_seen_cache_test.go

```diff
@@ -17,22 +17,27 @@ func TestFirstSeenCacheFound(t *testing.T) {
 }
 
 func TestFirstSeenCacheExpire(t *testing.T) {
+	backgroundSweepInterval = time.Second
+
 	tc := newFirstSeenCache(time.Second)
-	for i := 0; i < 11; i++ {
+	for i := 0; i < 10; i++ {
 		tc.Add(fmt.Sprint(i))
 		time.Sleep(time.Millisecond * 100)
 	}
 
+	time.Sleep(2 * time.Second)
```

Comment:
Would be nice if we could use a mock clock here.

Reply:
yeah... future pr ;)